res/values-tl/strings.xml

```diff
@@ -217,22 +217,22 @@
     <string name="GOAL_">"GOAL!:"</string>
     <string name="SOCCER">"SOCCER:"</string>
     <string name="DRAW">"PATAS:"</string>
-    <string name="Clan_Name">"Clan Name:"</string>
-    <string name="Clan">"Clan:"</string>
-    <string name="JOIN_CLAN">"SUMALI SA CLAN:"</string>
-    <string name="JOIN_RANDOM_CLAN">"SUMALI SA KAHIT ANONG CLAN:"</string>
+    <string name="Clan_Name">"Pangalan ng Angkan:"</string>
+    <string name="Clan">"Angkan:"</string>
+    <string name="JOIN_CLAN">"SUMALI SA ANGKAN:"</string>
+    <string name="JOIN_RANDOM_CLAN">"SUMALI SA KAHIT ANONG ANGKAN:"</string>
     <string name="LEAVE_CLAN">"UMALIS SA CLAN:"</string>
```

Comment:
this string contains the word "clan" and you did not translate it like the rest.